src/applications/claims-status/tests/e2e/page-objects/TrackClaimsPageV2.js

```diff
@@ -289,31 +297,49 @@ class TrackClaimsPageV2 {
     cy.axeCheck();
   }
 
-  submitFilesForReview() {
+  submitFilesForReview(isOldVersion = false) {
     cy.intercept('POST', `/v0/evss_claims/189685/documents`, {
       body: {},
     }).as('documents');
-    cy.get('.usa-file-input input')
-      .selectFile({
-        contents: Cypress.Buffer.from('test file contents'),
-        fileName: 'file-upload-test.txt',
-        mimeType: 'text/plain',
-        lastModified: Date.now(),
-      })
+    cy.get('#file-upload')
```

Comment:
Updated the logic to reference the correct class since `.usa-file-input` no longer exists